.travis.yml

```diff
@@ -41,13 +43,13 @@ matrix:
         osx_image: xcode9.4
         env: CONAN_APPLE_CLANG_VERSIONS=9.1
       - <<: *osx
-        osx_image: xcode10
+        osx_image: xcode10.1
         env: CONAN_APPLE_CLANG_VERSIONS=10.0
 
 install:
-  - chmod +x .travis/install.sh
-  - ./.travis/install.sh
+  - chmod +x .ci/install.sh
+  - ./.ci/install.sh
 
 script:
-  - chmod +x .travis/run.sh
-  - ./.travis/run.sh
+  - chmod +x .ci/run.sh
```

Comment:
note that you change this path but scripts are still in `.travis` folder

Reply:
Yeah, I made some shit here, it's all broken on CI :fire: